### PR TITLE
Fix #329 Use MUI v5

### LINKED ISSUE #329: (React / General) Use MUI v5
The latest version of MUI(material-ui) is v5 https://mui.com/blog/mui-core-v5/

### PR DESCRIPTION
**Pull request by syucream**
https://github.com/dmm-com/airone/issues/329

The latest version of MUI(material-ui) is v5 https://mui.com/blog/mui-core-v5/.

MUI v5 changes default palette so some theme colors are changed. We need to discuss having our palette.
image

### CODE REVIEW

File: frontend/src/pages/Entity.test.js [hunk 1]
```diff
@@ -46,18 +47,10 @@ test("should match snapshot", async () => {
...
 
   // wait async calls and get rendered fragment
-  const { container } = render(<Entity />, {
-    wrapper: MemoryRouter,
+  const result = render(<Entity />, {
+    wrapper: TestWrapper,
   });
   await waitForElementToBeRemoved(screen.getByTestId("loading"));
 
-  /**
```
Comment (syucream) [inline frontend/src/pages/Entity.test.js:54]: It's fixed since MUI v5.

File: frontend/src/utils/TestWrapper.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+
+export function TestWrapper({ children }) {
```
Comment (syucream) [inline frontend/src/utils/TestWrapper.js:5]: I define a wrapper render component for react testing library to inject needed context providers and etc easily.

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 import { ErrorHandler } from "./ErrorHandler";
 
+const theme = createTheme();
+
 function App({}) {
   return (
-    <ErrorHandler>
-      <AppRouter />
-    </ErrorHandler>
+    <ThemeProvider theme={theme}>
```
Comment (syucream) [inline frontend/src/App.js:12]: Provide a theme explicitly because MUI v5 is no longer providing default theme automatically.

File: package.json [hunk 4]
```diff
@@ -49,9 +48,11 @@
     "webpack-cli": "^4.4.0"
   },
   "dependencies": {
+    "@emotion/styled": "^11.6.0",
     "@jest/globals": "^27.0.6",
-    "@material-ui/core": "^4.12.3",
-    "@material-ui/lab": "^4.0.0-alpha.60",
+    "@mui/icons-material": "^5.2.1",
+    "@mui/material": "^5.2.3",
+    "@mui/styles": "^5.2.3",
```
Comment (userlocalhost) [inline package.json:55]: Excellent!!

**Review by userlocalhost — APPROVED** [review]
LGTM 👍

**Pull request merged**